doc/TESTING.md

```diff
@@ -1,48 +1,72 @@
 # Testing
-
-This chapter should provide the user with all needed information around testing within your project.
+This chapter provides the user with all needed information around testing within this project.
 
 ## Content
-
-- [Different kind of tests](#different-kind-of-tests)
-- [Tooling](#tooling)
-- [How to write tests](#how-to-write-tests)
-- [How to run tests](#how-to-run-tests)
+- [Testing](#testing)
+  - [Different kind of tests](#different-kind-of-tests)
+  - [Testing strategy](#testing-strategy)
+  - [Tooling](#tooling)
+     - [open-wc/testing](#open-wc/testing)
+  - [How to write tests](#how-to-write-tests)
+  - [How to run tests](#how-to-run-tests)
```

Comment:
Please also include "Content".

The link to open-wc/testing is not working. I the anchor would need to be "#open-wc-testing" for it to work.

Please also include "Contributing".

Reply:
Adjusted now.